pkg/apb/executor.go

```diff
@@ -153,6 +153,11 @@ func ExecuteApb(
 
 	log.Notice(fmt.Sprintf("Creating pod %q in the %s namespace", pod.Name, executionContext.Namespace))
 	_, err = k8scli.Client.CoreV1().Pods(executionContext.Namespace).Create(pod)
+	if err != nil {
+		log.Errorf("Failed to create pod - %v", err)
+		return executionContext, err
```

Comment:
Redundant return statement. The error will get bubbled up and should get checked by executor callers, but this is probably still valuable for debug reasons, it just might result in some louder logs, which seems fine to me.

Reply:
I'm going to remove the if block entirely since all of our calls to the ExecuteAPB look something like:

```
	executionContext, err := ExecuteApb(
...
	)
        defer runtime.Provider.DestroySandbox(...)
	if err != nil {
		log.Errorf("Problem executing apb [%s] $ACTION_NAME", executionContext.PodName)
		return executionContext.PodName, nil, err
	}
```